client/src/components/PitContent.js

```diff
@@ -1,12 +1,109 @@
-import React, {Component} from 'react'
+import React, { Component } from "react";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import "./PitContent.css";
 
 class PitContent extends Component {
+  state = {
```

Comment:
Same comment here. I think a constructor might be a good practice. Let's check with Jon.

Reply:
same as above. Removing constructor is ok as default super is called, but let's make sure of the convention for setting initial state properties.